stakedb/ticketpool.go

```diff
@@ -293,28 +230,19 @@ func NewTicketPool(dataDir, dbSubDir string) (tp *TicketPool, err error) {
 	err = db.RunValueLogGC(rewriteThreshold)
 	if err != nil {
 		if err != badger.ErrNoRewrite {
-			return nil, fmt.Errorf("failed badger.RunValueLogGC: %v", err)
+			return nil, fmt.Errorf("failed badger.RunValueLogGC: %w", err)
 		}
 		log.Debugf("badger value log not rewritten (OK).")
 	}
 
-	// Load all diffs
+	// Load all diffs.
 	log.Infof("Loading all ticket pool diffs from DB version %d...", ver)
-	poolDiffs, heights, err := LoadAllPoolDiffs(db, ver)
+	poolDiffs, _, err := loadAllPoolDiffs(db)
 	if err != nil {
```

Comment:
@chappjc, `heights` is not used anywhere other than this file, or am I missing something?